lib/set-schema-options.js

```diff
@@ -1,5 +1,9 @@
+const keysToIgnore = ["customOpts"];
+
 module.exports = function(schema, options) {
   for (var key in options) {
+    if (keysToIgnore.includes(key)) continue;
+
     if (options.hasOwnProperty(key)) {
```

Comment:
combine these 2 conditions? 
```suggestion
    if (!keysToIgnore.includes(key) && options.hasOwnProperty(key)) {
```

Reply:
updated